apps/web/src/pages/Pools/PoolAdvertisementPage/components/EducationRequirements.tsx

```diff
@@ -0,0 +1,128 @@
+            id: "bHHnHZ",
+            description:
+              "Descriptive text explaining valid applied work experiences",
+          })}
+        </Text>
+        <ul>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "On-the-job learning",
+              id: "qNL/Rp",
+              description: "pool experience requirement, on job learning",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "Non-conventional training",
+              id: "YlWJ/N",
+              description:
+                "pool experience requirement, non-conventional training",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "Formal education",
+              id: "DydUje",
+              description: "pool experience requirement, formal education",
+            })}
+          </li>
+          <li>
+            {intl.formatMessage({
+              defaultMessage: "Other field related experience",
+              id: "GNvz2K",
+              description: "pool experience requirement, other",
+            })}
+          </li>
+        </ul>
+      </RequirementCard>
+      <span
+        aria-hidden="true"
+        data-h2-background-color="base(white)"
+        data-h2-display="base(flex)"
+        data-h2-align-items="base(center)"
+        data-h2-justify-content="base(center)"
+        data-h2-height="base(x2)"
+        data-h2-width="base(x2)"
+        data-h2-radius="base(x2)"
+        data-h2-shadow="base(s)"
+        data-h2-font-weight="base(700)"
+        data-h2-align-self="base(center)"
+        data-h2-text-transform="base(uppercase)"
+        data-h2-position="l-tablet(absolute)"
+        data-h2-top="l-tablet(50%)"
+        data-h2-left="l-tablet(50%)"
+        data-h2-transform="l-tablet(translate(-50%, -50%))"
+        data-h2-margin="base(-x1.25 auto) l-tablet(0)"
+        data-h2-z-index="base(2)"
+      >
+        {intl.formatMessage({
+          defaultMessage: "or",
+          id: "l9AK3C",
+          description:
+            "that appears between different experience requirements for a pool applicant",
+        })}
+      </span>
+      <RequirementCard>
+        <Heading level="h4" size="h6" data-h2-margin-top="base(0)">
+          {intl.formatMessage({
+            defaultMessage: "2-year post-secondary",
+            id: "ZIwaDE",
+            description: "Title for the education requirements",
+          })}
+        </Heading>
+        <Text>
+          {intl.formatMessage({
+            defaultMessage:
+              "Successful completion of two years of post secondary education in computer science, information technology, information management or another specialty relevant to this position.",
```

Comment:
@substrae post-secondary should be hyphenated or one word but never two words.

REF: https://www.canada.ca/en/treasury-board-secretariat/services/staffing/qualification-standards/core.html#:~:text=Graduation%20from%20a%20two%2Dyear%20program%20of%20study%20from%20a%20recognized%20post%2Dsecondary%20institution%20with%20acceptable%20specialization%20in%20computer%20science%2C%20information%20technology%2C%20information%20management%20or%20another%20specialty%20relevant%20to%20the%20position%20to%20be%20staffed.

This spelling also appears on the Pool Advertisement Page:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/091306b9e0196af86c925d6a662c7653cb3182d1/apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx#L681-L687

Reply:
> post-secondary should be hyphenated or one word but never two words.

@mnigh Fixed as part of [update copy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6543/commits/205318329b8dce1892011f38b3846615fe6609fd)

Reply:
@esizer it looks like you missed changing the id for `r9FSaq` in French though 😉. 

also, i think a related term got translated incorrectly in French recently as it should be postsecondaire not post-secondaire:
https://github.com/GCTC-NTGC/gc-digital-talent/blob/091306b9e0196af86c925d6a662c7653cb3182d1/packages/i18n/src/lang/fr.json#L1373-L1376

Reply:
> @esizer it looks like you missed changing the id for `r9FSaq` in French though 😉. 
> 
> also, i think a related term got translated incorrectly in French recently as it should be postsecondaire not post-secondaire:
> https://github.com/GCTC-NTGC/gc-digital-talent/blob/091306b9e0196af86c925d6a662c7653cb3182d1/packages/i18n/src/lang/fr.json#L1373-L1376


Did not forget, waiting on new translations to update it 👍